lang/py3/tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
```

Comment:
I would not include `tox`. Personally, I strongly feel that tox doesn't add any value anymore. It is from the pre-Docker time in which it was hard to isolate Python versions.

Reply:
I used Tox because it is still simpler than changing all the tooling to support docker

Reply:
There is already support for Docker. Having Tox inside of docker does not make any sense since they are both for isolating the application from the system.